ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/ParticipantConfig.scala

```diff
@@ -33,4 +35,7 @@ object ParticipantConfig {
 
   // this pool is used for all data access for the ledger api (command submission, transaction service, ...)
   val DefaultApiServerDatabaseConnectionPoolSize = 16
+
+  val DefaultMaxContractStateCacheSize: Long = 100000
```

Comment:
What dimension does this number have? Is this in bytes, megabytes or does it represent number of items or something else?

Reply:
It's maximum number of entries in the cache.

Reply:
@miklos-da If you asked, it means it is ambiguous. I'll add a note to make it clear when we promote the feature from experimental to production-grade